server/test/ping.spec.js

```diff
@@ -1,22 +1,26 @@
-const chai = require("chai");
-const chaiHttp = require("chai-http");
-const app = require("../app.js");
-
-chai.should();
-chai.use(chaiHttp);
-
-describe("/POST ping", () => {
-  it("it should return 200 and message", (done) => {
-    chai
-      .request(app)
-      .post(`/ping/`)
-      .send({ message: "hello" })
-      .end((err, res) => {
-        res.should.have.status(200);
-        res.body.should.have
-          .property("response")
-          .eql("Server is running. Message received: hello");
-        done();
-      });
+var expect = require('chai').expect;
+var request = require('request');
+
+
+describe("profile GET route", function () {
+
+  var url = "http://localhost:3001/profiles/profiles"
```

Comment:
I would prefer a const being used here just for convention in the overall project.